client/components/plans/plans-compare/index.jsx

```diff
@@ -117,15 +112,15 @@ var PlansCompare = React.createClass( {
 		}
 	},
 
-	isSubmitting: function() {
+	isSubmitting() {
 		if ( ! this.props.transaction ) {
 			return false;
 		}
 
-		return this.props.transaction.step.name === transactionStepTypes.SUBMITTING_WPCOM_REQUEST
+		return this.props.transaction.step.name === SUBMITTING_WPCOM_REQUEST
```

Comment:
Yep I think it looks fine in this scenario! We just need to be aware than it can be ambiguous in some cases though.